webpack.dev.config.js

```diff
@@ -35,6 +35,7 @@ var paths = [
   '/components/tabs/',
   '/components/jumbotron/',
   '/components/alerts/',
+  '/components/collapse',
```

Comment:
can you add a trailing `/` to generate the correct route path. I noticed a console warning regarding the active class applied to the nav .